_sass/_base.scss

```diff
@@ -85,52 +85,55 @@ h3,
 h4,
 h5,
 h6 {
-  color: $primary-font-color-dark;
+  color: $heading-font-color;
   font-family: $heading-font;
   font-weight: $heading-font-weight;
-  line-height: line-height(2);
   margin: line-height(2) 0 0 line-height(-1);
```

Comment:
While we're at it, can we get rid of this right margin on the header elements?

![image](https://cloud.githubusercontent.com/assets/4473327/24880722/4bfd2572-1df0-11e7-8f97-0658290ccbfd.png)